spec/integration/commands/update_spec.rb

```diff
@@ -27,6 +27,25 @@ def by_name(name)
     relation.insert(name: 'Piotr')
   end
 
+  context '#transaction' do
+    it 'update record if there was no errors' do
+      result = users.update.transaction do
+        users.update.by_id(piotr[:id]).set(peter)
+      end
+
+      expect(result.value).to eq([{ id: 1, name: 'Peter' }])
+    end
+
+    it 'updates nothing if error was raised' do
+      users.update.transaction do
+        users.update.by_id(piotr[:id]).set(peter)
+        raise Sequel::Rollback
+      end
```

Comment:
Should we introduce `ROM::SQL::Rollback` or use `throw/catch` and do `throw :rollback`?